floris/floris.py

```diff
@@ -19,6 +19,7 @@ class Floris():
 
     inputs:
         input_file: str - path to the json input file
+        input_dict: str - dict of appropriate inputs
```

Comment:
I believe this line should be

```suggestion
        input_dict: dict - dictionary of appropriate inputs
```

Reply:
Good catch, I fixed this in another commit